infrastructure/init.sh

```diff
@@ -96,15 +104,19 @@ fi
 
 
 # Create ACM Certificate that will be used in Route 53 Domain
-if aws acm request-certificate \
-    --domain-name "${domain}" \
-    --validation-method DNS \
-    --key-algorithm RSA_2048 \
-    --region "${aws_region}" \
-    --tags "[{\"Key\":\"Project\",\"Value\":\"${project}\"}, {\"Key\":\"Owner\",\"Value\":\"${owner_email}\"}, {\"Key\":\"Creator\",\"Value\":\"${creator_email}\"}]" >/dev/null 2>&1; then
-    echo "ACM certificate requested successfully"
+if [[ -z "${domain}" ]]; then
+    echo "Domain is null. Skipping ACM certificate request."
 else
-    echo "Failed to request ACM certificate"
+    if aws acm request-certificate \
+        --domain-name "${domain}" \
+        --validation-method DNS \
+        --key-algorithm RSA_2048 \
+        --region "${aws_region}" \
+        --tags "[{\"Key\":\"Project\",\"Value\":\"${project}\"}, {\"Key\":\"Owner\",\"Value\":\"${owner_email}\"}, {\"Key\":\"Creator\",\"Value\":\"${creator_email}\"}]" >/dev/null 2>&1; then
+        echo "ACM certificate requested successfully"
+    else
+        echo "Failed to request ACM certificate"
+    fi
 fi
 
 # Create the KMS key for cloudtrail and capture the key ID in a variable
```

Comment:
now KMS key should be removed from init.sh

Reply:
added in c250f7e1a2865da183d659363fa7e938f01ac440